ci/dependency-update/Jenkinsfile

```diff
@@ -42,7 +42,7 @@ Map settings() {
 					updateProperties: [],
 					onlyRunTestDependingOn: ['hibernate-search-backend-elasticsearch'],
 					// We want to use the snapshot version of an image from the ES registry since that's where they are publishing their snapshots.
-					additionalMavenArgs: '-Dtest.elasticsearch.run.elastic.image.name=docker.elastic.co/elasticsearch/elasticsearch -Dtest.elasticsearch.run.elastic.image.tag=master-SNAPSHOT',
+					additionalMavenArgs: '-Dtest.elasticsearch.run.elastic.image.tag=master-SNAPSHOT',
```

Comment:
I'm not sure I like this change, as future me will most likely end up reading this code and wonder "but, but, if I run `docker pull elastic/elasticsearch:master-SNAPSHOT` it doesn't work!?!".

Any reason you didn't want to follow my suggestion of just... not trying to pull images that are not from dockerhub before the build?

Reply:
well ... I wanted to try and have something more generic that we can potentially reuse if we start pulling images from other registries (e.g. maybe at some point from amazon for OpenSearch etc...) and ` if (containerImageRef.startsWith('docker.elastic.co/') ) { continue }` didn't look as something very scalable in those terms 😃